docs/HowToCertificatesRotation.md

```diff
@@ -0,0 +1,473 @@
+    --alias ca_prime --cert /tmp/ca.cert --timeout 30
+```
+
+The `--timeout` argument specifies the amount of time to wait for the new certificate to be propagated to the truststore of every process in the domain.
+
+Ensure that the new CA certificate with alias `ca_prime` is trusted by all APs and database processes as described in [Verify Domain Certificates](#verify-domain-certificates).
+
+Add the new CA certificate to the truststore of every SQL client.
+
+Generate and sign the new server certificates using the new CA certificate chain as described in [Rotate Server Certificate](#rotate-server-certificate).
+
+### Rotate Server Certificate
+
+Generate a new server key pair:
+
+```bash
+kubectl exec -ti generate-nuodb-certs -- \
+  nuocmd create keypair \
+    --keystore /tmp/keys/nuoadmin.p12 --store-password "$PASSWD" --ca \
+    --dname "CN=*.nuodb.svc.cluster.local, OU=Eng, O=NuoDB, L=Boston, ST=MA, C=US, SERIALNUMBER=67890"
+```
+
+In the example above, we specify the `SERIALNUMBER` field so that the new certificate has a different distinguished name from the current certificate.
+
+The new server certificates should be signed by the CA certificate.
+If the TLS keys have been generated using the `setup-keys.sh` script, the private/public key pair of the CA are available in the `ca.p12` file.
+In case the CA certificate has been renewed right now, use the new CA keystore generated using the [Rotate CA Certificate](#rotate-ca-certificate) steps.
+
+Generate and sign the new server certificates using the CA certificate chain:
+
+```bash
+kubectl cp ca.p12 generate-nuodb-certs:/tmp/keys/
+
+kubectl exec -ti generate-nuodb-certs -- \
+  nuocmd sign certificate \
+    --keystore /tmp/keys/nuoadmin.p12 --store-password "$PASSWD" \
+    --ca-keystore /tmp/keys/ca.p12 --ca-store-password "$PASSWD" \
+    --validity 36500 --ca --update
+```
+
+The `--ca` argument specifies whether the `isCA` extension is enabled in the generated certificate.
+This allows the admin to act as an intermediate CA and is used in the [Intermediate CA](./HowtoTLS.md#intermediate-ca) model.
+You can omit this argument in case the [Pass-down](./HowtoTLS.md#pass-down) model is used.
+
+The newly created keystore files can be copied on the client machine which will be used later to update the keystore secret.
+
+```bash
+kubectl cp generate-nuodb-certs:/tmp/keys/. /tmp/keys
+```
+
+If the server certificates are signed by a public CA, then follow the official steps to renew your certificates.
+
+> **NOTE**: For information on how to create the server keystore when using certificates signed by a Public CA, see [here](https://doc.nuodb.com/nuodb/latest/deployment-models/physical-or-vmware-environments-with-nuodb-admin/domain-operations/enabling-tls-encryption/using-certificates-signed-by-a-public-certificate-authority/).
+
+Update domain keystore with the renewed certificates as described in section [Update Key Pair Certificates](#update-key-pair-certificates).
```

Comment:
This line about referring to _Update Key Pair Certificates_ is omitted from the _Rotate CA Certificate_ section

Reply:
At the end of the _Rotate CA Certificate_ there is a line indicating that all steps in the `Rotate Server Certificate` section should be executed which includes _Update Key Pair Certificates_.